index.js

```diff
@@ -1,30 +1,44 @@
 const score=document.querySelector(".score")
-let timeLeft=document.querySelector(".time-left")
+const timeLeft=document.querySelector(".time-left")
 const mole=document.querySelectorAll(".mole")
 const box=document.querySelectorAll(".box")
 
 let hitid=null;
+let totalScore=0;
 
-//remove the presence of class mole
+//initially remove the presence of any class mole 
```

Comment:
Both the previous and updated comments are confusing. What does `class mole` mean? Is there a class called `mole`? And what does `remove the presence` mean? Does that mean removing a class completely?